src/dos/program_imgmount.cpp

```diff
@@ -212,13 +212,20 @@ void IMGMOUNT::Run(void) {
         return;
     }
 
+    std::vector<std::string> files;
     // find all file parameters, assuming that all option parameters have been removed
     while (cmd->FindCommand((unsigned int)(paths.size() + 2), temp_line) && temp_line.size()) {
         // Try to find the path on native filesystem first
         const std::string real_path = to_native_path(temp_line);
         if (real_path.empty()) {
-            LOG_MSG("IMGMOUNT: Path '%s' not found, maybe it's a DOS path",
-                    temp_line.c_str());
+            if (get_expanded_files(temp_line, files, true)) {
```

Comment:
Try to avoid having functions that take in bools, when you see `true` or `false` passed into a function it doesn't help readers and often means there are separate code path - like a master switch for a function.  https://alexkondov.com/should-you-pass-boolean-to-functions/

In this case, at a minimum, let's put the `true` in a `constexpr` variable with a name that says whatever this bool is doing to the function.  Maybe something like, `constexpr auto only_expand_files = true`.

Reply:
Added constexpr as suggested.